modules/corelib/table.lua

```diff
@@ -197,6 +197,13 @@ function table.collect(t, func)
     return res
 end
 
+function table.insertall(t, s)
```

Comment:
```suggestion
function table.insertAll(t, s)
```

Reply:
`insertAll` breaks the table convention of lowercase method calls. We can leave it lower case until they are all changed later to match the convention I think